moveit_core/distance_field/include/moveit/distance_field/voxel_grid.h

```diff
@@ -61,6 +62,8 @@ template <typename T>
 class VoxelGrid
 {
 public:
+  MOVEIT_DECLARE_PTR_MEMBER(VoxelGrid);
+
```

Comment:
Well, `VoxelGrid<T>::VoxelGridPtr` is a bit long. `VoxelGrid<T>::Ptr` is also closer to the non-template case.